types/domexception/webidl2js-wrapper.d.ts

```diff
@@ -1,66 +0,0 @@
-import { Options as WebIDLConversionOptions } from 'webidl-conversions';
```

Comment:
Deleting this file is incorrect

Reply:
Yup, I misunderstood the package. Fixed now.